README.md

```diff
@@ -66,6 +72,19 @@ All `Meter`s are reporting in `bytes`.
 
 * `process.threads`: The number of process threads as seen by the operating system.
 
+### ContextSwitchMetrics
+
+`ContextSwitchMetrics` reads process-level context switch information from `/proc/self/status`.
+
+> Please note that `procfs` is only available on Linux-based systems.
+
+* `process.voluntary.context.switch`: The accumulative number of voluntary context switches since application start.
+  Voluntary context switches happens when the thread is in waiting or blocked state, and scheduler switches control
+  to a different thread.
```

Comment:
A voluntary context switch occurs when a thread is in a waiting or blocked state and the scheduler switches control to another thread.